opentech/apply/stream_forms/blocks.py

```diff
@@ -158,6 +158,7 @@ def __init__(self, *args, **kwargs):
             'data-date-format': 'yyyy-mm-dd',
         })
         kwargs['attrs'] = attrs
+        kwargs['format'] = '%Y-%m-%d'
```

Comment:
Why not make this site-wide using `WAGTAIL_DATE_FORMAT` 
 - http://docs.wagtail.io/en/v1.13.1/topics/streamfield.html#dateblock

Reply:
could be useful to configure both date and datetime as per http://docs.wagtail.io/en/v1.13.1/advanced_topics/settings.html#date-and-datetime-inputs

Reply:
Because we aren't in wagtail anymore

`Specifies the date and datetime format to be used in input fields in the Wagtail admin`

The mechanism we are actually using is:
https://github.com/django/django/blob/1.11.9/django/forms/widgets.py#L468
which _should_ fall through to using this value: https://github.com/django/django/blob/1.11.9/django/utils/formats.py#L22
(which is the same as the value I included above)

As i said i think the `formats.localize_input` is messing with this but didn't get to the bottom of it. 

I'll spend a bit more time looking at this when we know what the format required from  OTF is.

Reply:
fair enough. What I am trying to get at is this should be consistent throughout, so we need to tie it in with the wagtail date formats one way or another when we get to it